components/reusable-components/navigation/Navigation.tsx

```diff
@@ -24,28 +25,44 @@ const Navigation = () => {
         <Link className={styles.navigationLogoLink} href="/">
           <Image src={LogoNavigation} className={styles.navigationLogo} alt="LearnHub Logo" />
         </Link>
-        <div className={styles.toggleContainer} onClick={toggleTheme}>
-          <div className={`${styles.animate} ${!isSun ? styles.moveRight : ''}`}>
-            <div className={`${styles.toggleButton} ${isSun && styles.marginLeftButton}`} />
-          </div>
 
-          {isSun ? (
-            <Image
-              src={moonImage}
-              alt="moon icon"
-              width={18}
-              height={18}
-              className={styles.moonImage}
-            />
-          ) : (
-            <Image
-              src={sunImage}
-              alt="sun icon"
-              width={18}
-              height={18}
-              className={styles.sunImage}
-            />
-          )}
+        <div className={styles.navigationActions}>
+          <Button
+            href="/register"
+            type="link"
+            buttonClass={['primaryButton', 'navigationButton', 'button']}
+            buttonText="Регистрирај се"
+          />
+          <Button
+            href="/login"
+            type="link"
+            buttonClass={['secondaryButton', 'navigationButton', 'button']}
+            buttonText="Најави се"
+          />
```

Comment:
@MarijaTrajkoska can you revert the changes made in the navigation(adding the buttons)?
We will use this buttons only for the platform. So we will add them again when we are implementing the functionalities regarding the platform